WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/Cells/RefundCustomAmountsDetailsTableViewCell.swift

```diff
@@ -68,13 +68,14 @@ extension RefundFeesDetailsTableViewCell {
 
 // MARK: Constants
 
-private extension RefundFeesDetailsTableViewCell {
+private extension RefundCustomAmountsDetailsTableViewCell {
     enum Localization {
         static let taxTitle = NSLocalizedString(
             "Tax", comment: "Title on the refunds screen that lists the fees tax cost")
         static let subtotalTitle = NSLocalizedString(
             "Subtotal", comment: "Title on the refund screen that lists the fees subtotal cost")
-        static let totalTitle = NSLocalizedString(
-            "Fees Refund", comment: "Title on the refund screen that lists the fees total cost")
+        static let totalTitle = NSLocalizedString("refundIssue.customAmounts.totalTitle",
```

Comment:
Question: When using the reverse-DNS key pattern for localized strings, should we follow any specific convention for the rest of the string that is not the variable name? 

As an example, when reading the P2 ( p91TBi-aJl-p2 ) I would expect something like `refundCustomAmountsDetails.totalTitle`

Reply:
Thanks for the comment! I think there's no exact convention for this, but I used your suggestion in 4693b7b1280454e33e0603fe8c6d0d24589c7ece